src/library/components/signup/select_input.js

```diff
@@ -16,9 +14,8 @@ var SelectInput = function() {
       return this.props.onChange(option);
     },
     render: function() {
-      var className, disabled, loadOptions, placeholder, ref1;
-      ref1 = this.props, placeholder = ref1.placeholder, loadOptions = ref1.loadOptions, disabled = ref1.disabled;
-      className = 'select-input';
+      var ref1 = this.props, placeholder = ref1.placeholder, loadOptions = ref1.loadOptions, disabled = ref1.disabled,
+        className = 'select-input';
```

Comment:
A lot of this signup code was generated from the .js.coffee code in the rigse repo.
Some of it was manually corrected to make some assignments more readable and some of it wasn't. (Hence `var ref = React.DOM`, `var ref1 = ...`, etc.)
I am not sure if that's what is happening here, but shouldn't `className` be declared somewhere within function scope as
`var className` prior to this assignment `className = 'select-input';` ?

Reply:
className is part of the list of variable being declared by the single `var` keyword. It is separated by a comma from the line above, so this var keyword applies to it as well.

technically this change wasn't necessary to make jshint happy.  The actual problem was the statement on the old line 20:
`ref1 = this.props, placeholder = ref1.placeholder, loadOptions = ref1.loadOptions, disabled = ref1.disabled;` 
It isn't normal to separate statements by commas. Apparently browsers can handle it, but it is not recommended.